### ISSUE TITLE
why is my loss so high ? and my val_loss is nan? 

### ISSUE THREAD

**Issue by phamngocthanhtrung** [qqwweee/keras-yolo3#304]
hi guys, 
i configurate my yolov3_1c.cfg with 
Line 3: set batch=24, this means we will be using 24 images for every training step
Line 4: set subdivisions=8, the batch will be divided by 8 to decrease GPU VRAM requirements.
Line 603: set filters=(classes + 5)*3 in our case filters=15
Line 610: set classes=1, the number of categories we want to detect
Line 689: set filters=(classes + 5)*3 in our case filters=15
Line 696: set classes=1, the number of categories we want to detect
Line 776: set filters=(classes + 5)*3 in our case filters=15
Line 783: set classes=1, the number of categories we want to detect

and i get the following output, here you can see that my loss is very high and my val loss ist nan. 
![colab_vallossnann](https://user-images.githubusercontent.com/44701408/50157041-a1a66300-02d0-11e9-9b31-6421c8fcdb8b.PNG)

i have read that i should to change my code referenced on this issue #79 
but i downloaded the repository a few days ago 
i have checked, that my code is updated with this reference #79 
please help me :) 
best regards

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by phamngocthanhtrung** [comment]
.

**Comment by Hazelsuko07** [comment]
Same issue

**Comment by S601327412** [comment]
> hi guys,
> i configurate my yolov3_1c.cfg with
> Line 3: set batch=24, this means we will be using 24 images for every training step
> Line 4: set subdivisions=8, the batch will be divided by 8 to decrease GPU VRAM requirements.
> Line 603: set filters=(classes + 5)*3 in our case filters=15
> Line 610: set classes=1, the number of categories we want to detect
> Line 689: set filters=(classes + 5)*3 in our case filters=15
> Line 696: set classes=1, the number of categories we want to detect
> Line 776: set filters=(classes + 5)*3 in our case filters=15
> Line 783: set classes=1, the number of categories we want to detect
> 
> and i get the following output, here you can see that my loss is very high and my val loss ist nan.
> ![colab_vallossnann](https://user-images.githubusercontent.com/44701408/50157041-a1a66300-02d0-11e9-9b31-6421c8fcdb8b.PNG)
> 
> i have read that i should to change my code referenced on this issue #79
> but i downloaded the repository a few days ago
> i have checked, that my code is updated with this reference #79
> please help me :)
> best regards

why your's batch_size is only 32 but per step is so slowly?

**Comment by younghuvee** [comment]
I trained the model without pretrain weights to solved this issue.

**Comment by sxyxf66** [comment]
Why the loss is so big? Same issue. The loss is right?

**Comment by remcova** [comment]
> I trained the model without pretrain weights to solved this issue.

How did you do this? Can you explain? I'm currently having this issue as well

**Comment by vuthede** [comment]
@phamngocthanhtrung  have you resolved the problem yet?

**Comment by remcova** [comment]
Guys I recommend to reduce your batch size if you get a crazy high loss. Batch size 32 is pretty big and not suitable for some projects.

**Comment by vuthede** [comment]
thanks your suggestion @remcova .
Actually, now I try to train with only **one sample** with batchsize =1 to see if the model can overfit this sample or not. But the  train_loss is converged at  ~ 5.00, I am trying to manually reduce the learning rate but it didn't help. 
Am I right to say the train loss should be in range [0,1]
Do you have any idea?

**Comment by remcova** [comment]
@vuthede Makes sense but I'm not sure if it's correct. It's weird that the train_loss is converged at 5.00. If you want to know how well your model performs, I would recommend to use other metrics for example the F-measure (F1-score). That's what i'm using currently. It would be nice if @qqwweee takes a look at this issue.

**Comment by datduonguva** [comment]
I posted my solution in here. Hope it would help
https://github.com/qqwweee/keras-yolo3/issues/171#issuecomment-522287613
